packages/core/integration-tests/test/elm.js

```diff
@@ -1,59 +1,89 @@
 import assert from 'assert';
-import {bundle, assertBundleTree, run, outputFS} from '@parcel/test-utils';
+import path from 'path';
+import {
+  bundle,
+  distDir,
+  assertBundles,
+  run,
+  outputFS,
+} from '@parcel/test-utils';
 
-describe.skip('elm', function() {
+describe('elm', function() {
   it('should produce a basic Elm bundle', async function() {
-    let b = await bundle(__dirname + '/integration/elm/index.js');
+    let b = await bundle(path.join(__dirname, '/integration/elm/index.js'));
 
-    await assertBundleTree(b, {
-      type: 'js',
-      assets: ['Main.elm', 'index.js'],
-    });
+    await assertBundles(b, [
+      {
+        type: 'js',
+        assets: ['Main.elm', 'index.js'],
+      },
+    ]);
 
     let output = await run(b);
     assert.equal(typeof output().Elm.Main.init, 'function');
   });
   it('should produce a elm bundle with debugger', async function() {
-    let b = await bundle(__dirname + '/integration/elm/index.js');
+    let b = await bundle(path.join(__dirname, '/integration/elm/index.js'));
 
     await run(b);
-    let js = await outputFS.readFile(__dirname + '/dist/index.js', 'utf8');
+    let js = await outputFS.readFile(path.join(distDir, 'index.js'), 'utf8');
     assert(js.includes('elm$browser$Debugger'));
   });
 
   it('should apply elm-hot if HMR is enabled', async function() {
-    let b = await bundle(__dirname + '/integration/elm/index.js', {
-      hmr: true,
+    let b = await bundle(path.join(__dirname, '/integration/elm/index.js'), {
+      hot: true,
     });
 
-    await assertBundleTree(b, {
-      type: 'js',
-      assets: ['Main.elm', 'hmr-runtime.js', 'index.js'],
-    });
+    await assertBundles(b, [
+      {
+        type: 'js',
+        assets: [
+          'HMRRuntime.js',
+          'Main.elm',
+          'index.js',
+          'react-refresh-runtime.development.js',
+          'ReactRefreshRuntime.js',
```

Comment:
Can you add these two files (containing `{}` and being empty, respectively):
```
	packages/core/integration-tests/test/integration/elm/package.json
	packages/core/integration-tests/test/integration/elm/yarn.lock
```

Then the React related HMR assets aren't bundled here (as it should be).

Reply:
Sorry I don't quite understand what you mean by adding those two files? Should they be added in the `assertBundles` list or in `assets` from the `js`?

Reply:
No, sorry 😄 . Create these files and commit them. This will make Parcel use `packages/core/integration-tests/test/integration/elm` as the root for this test fixture (instead of the repo root, where the package.json contains React leading to these assets).